crates/ty_ide/src/completion.rs

```diff
@@ -5872,15 +5862,169 @@ from .imp<CURSOR>
             .auto_import()
             .module_names();
         assert_snapshot!(builder.build().snapshot(), @r"
-        Deprecated :: importlib.metadata
-        DeprecatedList :: importlib.metadata
-        DeprecatedNonAbstract :: importlib.metadata
-        DeprecatedTuple :: importlib.metadata
         deprecated :: typing_extensions
         deprecated :: warnings
         ");
     }
 
+    #[test]
+    fn reexport_simple_import_noauto() {
+        let snapshot = CursorTest::builder()
+            .source(
+                "main.py",
+                r#"
+import foo
+foo.ZQ<CURSOR>
+"#,
+            )
+            .source("foo.py", r#"from bar import ZQZQ"#)
+            .source("bar.py", r#"ZQZQ = 1"#)
+            .completion_test_builder()
+            .module_names()
+            .build()
+            .snapshot();
+        assert_snapshot!(snapshot, @"ZQZQ :: Current module");
```

Comment:
Probably orthogonal to this PR, but the "Current module" bit caught me by surprise here. `ZQZQ` isn't defined in the current module, it's defined in `bar.py`. But maybe we use this text just to indicate "not going to require a new import"? As in, `foo` has been imported into your current module, so `foo.ZQZQ` is already available here.

Reply:
Yeah this is just a test thing. But I agree the language could be clearer here. I changed this to `<no import required>`.